view/frontend/templates/js/replacejs.phtml

```diff
@@ -1400,14 +1410,33 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
         // When we know that cart will be updated soon we call configure with promises
         // to make sure bolt checkout with outdated cart isn't available for user
         $(document).on("ajax:addToCart", function () {
-            expectCartRendering = true;
-            // If merchant shows popup after add to cart
-            // delay 0 allows to render bolt button before we call configure
-            setTimeout(callConfigureWithPromises, 0);
-            // set flag to animate always present button when we catch cart updating
-            if (boltConfig.always_present_checkout) {
-                newItemAddedToCart = true;
+            var callBoltAfterAddToCart = function() {
+                expectCartRendering = true;
+                // If merchant shows popup after add to cart
+                // delay 0 allows to render bolt button before we call configure
+                setTimeout(callConfigureWithPromises, 0);
+                // set flag to animate always present button when we catch cart updating
+                if (boltConfig.always_present_checkout) {
+                    newItemAddedToCart = true;
+                }
             }
+            <?php
+            if ($isLoadConnectJsDynamic) {
+            ?>
+            if (insertConnectScriptWithoutSrc()) {
+                $.getScript("<?= /* @noEscape */ $connectJsUrl; ?>", function(data, textStatus, jqxhr) {
```

Comment:
Could you please explain why you prefer getScript instead of inserting script with src?

Reply:
Cause getScript has a callback function that is executed if the request succeeds.